app/assets/stylesheets/aspects.scss

```diff
@@ -4,20 +4,29 @@
     @include selectable-list;
 
     .status_indicator {
-      width: 19px;
+      width: 10px;
       height: 14px;
+
       display: inline-block;
     }
 
     a {
       .text {
-        color: #333333;
+        color: #33333;
```

Comment:
- Color literals like `#33333` should only be used in variable declarations; they should be referred to via variable everywhere else.
- Colors must have either three or six digits: `#33333`

Reply:
I assume a digit from this hex code was removed in error.